app/models/vc_facility_adapter.rb

```diff
@@ -51,8 +51,8 @@ def service_whitelist
   }.freeze
 
   HOURS_KEYMAP = %w[
-    Monday Tuesday Wednesday Thursday Friday Saturday Sunday
-  ].each_with_object({}) { |d, h| h[d] = d }
+    monday tuesday wednesday thursday friday saturday sunday
+  ].each_with_object({}) { |d, h| h[d.capitalize] = d }
```

Comment:
I don't think the `capitalize` is necessary since ultimately when we serialize this out of vets-api it gets converted to lowercase anyway. Don't we want a straight-through map from `monday => monday`, etc

Reply:
yea, true. I was mostly annoyed because my specs are inconsistent from one service to another. Are these all maintained by different groups? Why all the inconsistency? In a perfect world you would not need 4 distinct adapters.

Reply:
GeoBISL does all the ETL but the source data they have to work with varies wildly for all the different VBA/VHA/NCA groups feeding them data. There are some legitimate reasons for different adapters (like VHA facilities have medical services, others do not) but I decided it was not worth fighting over the minor consistency issues in the interest of not alienating people.